lib/msf/core/post/windows/services.rb

```diff
@@ -655,6 +663,94 @@ def meterpreter_service_list
 
           services
         end
+
+        #
+        # Start a service using sc.exe.
+        #
+        # @param name [String] Service name (not display name)
+        # @param server [String,nil] A hostname or IP address. Default is the
+        #   remote localhost.
+        #
+        # @return [Integer] 0 if service started successfully, 1 if it failed
+        #   because the service is already running, 2 if it is disabled
+        #
+        # @raise [RuntimeError] starting service failed
+        #
+        def _shell_service_start(service_name, server = nil)
+          host = server ? "\\\\#{server}" : nil
+          timeout = 62 # sc.exe default RPC connection timeout is 60 seconds
```

Comment:
```suggestion
          timeout = 75 # sc.exe default RPC connection timeout is 60 seconds plus the default 15s from cmd_exec
```
let's add a little more time just to be sure, also applies to _shell_service_stop

Reply:
changed